src/locksManager.js

```diff
@@ -47,14 +47,7 @@ var LocksManager = GObject.registerClass({
     }
 
     isUnlocked(lock) {
-        try {
-            const [variant] = this._proxy.GetSync(lock);
-            return !variant.deep_unpack()
-                           .locked
-                           .deep_unpack();
-        } catch (error) {
-            return false;
-        }
+        return !this._proxy.getDictValue(lock, 'locked', true);
```

Comment:
Seems like the code you are replacing had the default value equal to `false`, now it is `true`?

Reply:
@ptomato yes, and that is correct. !true === false, and that is what should return as default value (not the negation).